packages/shared/src/hooks/feed/useCommentPopup.ts

```diff
@@ -48,7 +51,11 @@ export default function useCommentPopup(feedName: string): {
             columns,
             column,
             row,
-            extra: { origin: 'feed', feed: feedName },
+            extra: {
+              origin: 'feed',
+              feed: feedName,
+              ...(ranking && { ranking }),
```

Comment:
Maybe we should create a global function that generates this `extra` object for the feed-based events?
It will be easier to keep track where it is used and that we keep the right format

Reply:
Great idea, working on that